### DISCUSSION TITLE
Edited branch but build not showing

### DISCUSSION THREAD

**Discussion by AbrarNoob** [ROM-builders/temporary#10336] [General]
Build not showing in cirrus after change done in branch...
https://github.com/ROM-builders/temporary/blob/alioth-ProjectBlaze-12.1-AbrarNoob/build_rom.sh

**Comment by Apon77** [comment]
> Build not showing in cirrus after change done in branch... https://github.com/ROM-builders/temporary/blob/alioth-ProjectBlaze-12.1-AbrarNoob/build_rom.sh

I don't see any problem like that
